com.unity.shadergraph/Editor/GraphUI/GraphElements/ModelUI/StaticPortParts/RadioPart.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEditor.ShaderGraph.GraphDelta;
+using UnityEditor.ShaderGraph.Registry;
+using UnityEditor.ShaderGraph.Registry.Types;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph.GraphUI
+{
+    public class RadioPart : SingleFieldPart<RadioButtonGroup, int>
+    {
+        protected override string UXMLTemplateName => "StaticPortParts/SliderPart";
+        protected override string FieldName => "sg-slider";
```

Comment:
RadioButtonGroup is used for SliderPart?

Reply:
Oops. Thanks. 
Updating

Reply:
I assume that there will be other errors of this format but I will change them all.

Reply:
Updated.